test/contexts/SignedInUserContext.test.jsx

```diff
@@ -59,12 +58,12 @@ describe('SignedInUserContext', () => {
     });
     fetchProfileInfo.mockResolvedValue({ profileInfo: {} });
     getPodUrlAll.mockResolvedValue(['https://example.com/pod/']);
-    const { container } = render(
+    const { findByText } = render(
       <SignedInUserContextProvider>
         <TestConsumer />
       </SignedInUserContextProvider>
     );
-    await flushPromises();
-    expect(container).toMatchSnapshot();
+    const val = await findByText('https://example.com/pod/');
```

Comment:
I've gotten a bit more familiar with RTL, and I've discovered that the `waitFor` and `findBy` functions do everything I was trying to do with `flushPromises`. This is in general a better approach, so I've removed `flushPromises`. This also removes the weird `act` warning message we were receiving in the test. RTL properly wraps these actions for us.

Reply:
Since `flushPromises` is no longer being used, and it was the only thing in the `testHelpers` file, I deleted the whole `testHelpers` file.